eq-author/src/apollo/createApolloErrorLink.js

```diff
@@ -1,13 +1,28 @@
+import {
+  setSentryUser,
+  setSentryTag,
+  sendSentryError,
+} from "../apollo/sentryUtils";
 import { onError } from "apollo-link-error";
 import { apiDownError } from "redux/saving/actions";
 import { signOutUser } from "../redux/auth/actions";
 
-export const errorHandler = (getStore, { networkError }) => {
-  if (networkError && networkError.bodyText === "User does not exist") {
-    getStore().dispatch(signOutUser());
-  } else if (networkError) {
+export const errorHandler = (getStore, error) => {
+  const { networkError, graphQLErrors } = error;
+  setSentryUser(window.localStorage.getItem("accessToken"));
+  if (networkError) {
     getStore().dispatch(apiDownError());
+    setSentryTag("networkError");
+    sendSentryError(networkError);
+    if (networkError.bodyText === "User does not exist") {
+      getStore().dispatch(signOutUser());
+    }
+    return;
   }
+  graphQLErrors.forEach(error => {
```

Comment:
I presume its an empty array when there is no problem?

Reply:
Yeah its just that if there is more than one graphql error it will come back as an array so had to loop through them

Reply:
If there are no errors the errorhandler wont be called

Reply:
Good point! 🤦‍♂